spec/stat_calculator_spec.rb

```diff
@@ -10,6 +10,9 @@
     @teams = Teams.all
     @game_teams = GameTeams.all
     @stat_calculator = StatCalculator.new(@games, @teams, @game_teams)
+    # Expected results - update these based csv files
+    @expected_winningest_coach = "Dan Lacroix"    # Ensure this is the coach with the highest win percentage
```

Comment:
These aren't necessary. We can put the string directly in the assertion statement. That way whoever reads the test does not have to scroll back and forth between these variables and the tests. Keeping them in the assertion statement also helps with readability.